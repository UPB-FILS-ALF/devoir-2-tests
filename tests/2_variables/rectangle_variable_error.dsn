& Rectangle with variables error
SET: %x1 20
SET: %y1 40
SET: %x2 150
RECTANGLE: %x1 %x2 %y1 %y2