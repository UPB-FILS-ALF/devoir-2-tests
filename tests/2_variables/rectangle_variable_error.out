ERROR LINE (5): Undefined variable y2
